frontend/src/components/readsets/ReadsetsListContent.tsx

```diff
@@ -389,61 +388,6 @@ function useColumns(filters: FilterSet, readsetTableCallbacks: PagedItemsActions
     return columns
 }
 
-function useExpandableMetricConfig(): ExpandableConfig<ObjectWithReadset> {
-    return useMemo(() => ({
```

Comment:
Any reason for getting rid of the useMemo ?

Reply:
sorry for the copy paste,

1. I dont think useMemo is necessary in this instance since it renders on a optionally filed variable (metric) with very few amounts of data in each.
2. I believe useMemo is to be used adequately, the key principle of useMemo is to memoize (remember) the data for the next render, in this instance, i didn't see any substantial benefit, it wouldn't offer any increase in performance when it comes down to this component.